tests/h/cli/commands/search_test.py

```diff
@@ -5,6 +5,11 @@
 
 from h.cli.commands import search
 
+pytestmark = [
+    pytest.mark.xdist_group("elasticsearch"),
+    pytest.mark.usefixtures("init_elasticsearch"),
+]
+
```

Comment:
**Unit** tests that use Elasticsearch also need this `xdist_group` thing that prevents Elasticsearch-based tests from being run in parallel with each other.